.github/workflows/devnet_deploy_contracts.yml

```
@@ -0,0 +1,30 @@
+name: Deploy devnet contracts
+
+on: workflow_dispatch
+
+jobs:
+  contracts:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          context: app
+
+      - name: Install Doppler CLI
+        uses: dopplerhq/cli-action@v1
+
+      - name: Deploy contracts
+        id: deploy-contracts
+        env:
+          DOPPLER_TOKEN: ${{ secrets.DEVNET_DOPPLER_TOKEN_CONTRACTS_DEPLOY }}
+        run: |
+          cd hardhat
+          npm ci
+            doppler run -- npx hardhat deploy --network devnet
+        
+      - name: Commit and push
+        run: |
+          git add .
+          git commit -m "Contract Migration Updated"
```

```suggestion
          git commit -m "chore: Contract Migration Updated"
```
To make this a conventional commit.